server/src/test/java/io/spine/server/event/DelegatingEventDispatcherShould.java

```diff
@@ -46,23 +58,28 @@ public void pass_null_tolerance_test() {
 
     @Test
     public void delegate_onError() {
-        final EmptyEventDispatcherDelegate delegate = new EmptyEventDispatcherDelegate();
-        final DelegatingEventDispatcher<String> delegatingDispatcher =
-                DelegatingEventDispatcher.of(delegate);
-
         final TestEventFactory factory = TestEventFactory.newInstance(getClass());
         EventEnvelope envelope = EventEnvelope.of(factory.createEvent(newUuidValue()));
 
-        delegatingDispatcher.onError(envelope, new RuntimeException("test delegating onError"));
+        final RuntimeException exception = new RuntimeException("test delegating onError");
+        delegatingDispatcher.onError(envelope, exception);
 
         assertTrue(delegate.onErrorCalled());
+        assertEquals(exception, delegate.getLastException());
     }
 
+    /*
+     * Test environment
+     */
```

Comment:
Either [this code style point](https://github.com/SpineEventEngine/core-java/wiki/Java-Code-Style#comment-separators-for-sections-of-the-class) is outdated, or this declaration should be updated accordingly.